py/desimeter/fieldmodel.py

```diff
@@ -92,12 +93,36 @@ def read_guide_stars_catalog(self,filename,max_sep_arcsec = 2.) :
         # here we could do some conversion of column names
         catalog = Table.read(filename)
 
+        if "mjd_obs" in catalog.dtype.names :
+            self.mjd = np.mean(catalog["mjd_obs"])
+            log.info("use mjd={} from catalog['mjd_obs']".format(self.mjd))
+
         if ( not  "xcentroid" in catalog.dtype.names ) or ( not "ra_gaia" in catalog.dtype.names ) :
             log.error("I can only deal with Aaron's catalogs with columns xcentroid,ycentroid,ra_gaia,dec_gaia, sorry")
             raise RuntimeError("I can only deal with Aaron's catalogs with columns xcentroid,ycentroid,ra_gaia,dec_gaia, sorry")
 
         max_sep_arcsec = 2.
```

Comment:
This is separate from my pm commit , but max_sep_arcsec is redefined here ; it was defined in keywords. On top of that it's not actually used as the condition down below just says <2 (instead of <max_sep_arcsec)